test/init.js

```diff
@@ -19,11 +13,11 @@ var config = require('rc')('loopback', {test: {mysql: {}}}).test.mysql;
 console.log(config);
 global.getConfig = function(options) {
   var dbConf = {
-    host: process.env.TEST_MYSQL_HOST || config.host || 'localhost',
-    port: process.env.TEST_MYSQL_PORT || config.port || 3306,
+    host: process.env.MYSQL_HOST || config.host || 'localhost',
```

Comment:
Should include the `TEST_` variants as well:
`host: process.env.TEST_MYSQL_HOST || process.env.MYSQL_HOST || config.host || 'localhost',`